restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/Conference.java

```diff
@@ -553,6 +572,12 @@ private void onStopRecording(StopRecording message, ActorRef self, ActorRef send
         }
     }
 
+    private void onReceiveTimeout(ReceiveTimeout message, ActorRef self, ActorRef sender) throws Exception {
+        if (logger.isInfoEnabled()) {
+            logger.info("Conference Received Timeout, will stop conference now.");
+        }
+        onStopConference(new StopConference(SUPER_ADMIN_ACCOUNT_SID), self, sender);
```

Comment:
@maria-farooq Maybe we should differentiate the StopConference initiated by SuperAdmin and the one initiated by the timer. This is just an idea, not sure about the implementation but I think if it sounds feasible to you we should investigate it

Reply:
we can provide a `reason` field in `StopConference` as well in API,  in case of timeout its will be `conference time limit reached` this field will later be passed as reason header in corresponding BYE to sip clients

Reply:
@maria-farooq sounds good